### ISSUE TITLE
crash in latest master when calling add() on iOS 8.1

### ISSUE THREAD

**Issue by RangerRick** [katzer/cordova-plugin-local-notifications#380]
I'm sending a notification with the options:

```javascript
{
  autoCancel=true,
  icon="file://images/monkey.png",
  sound=null,
  id=22,
  message="New Seamail: Hey, you're still an asshole.",
  badge=22,
  json="{\"id\":\"54b6a2fda7e46af7b900000a\",\"users\":[{\"username\":\"rangerrick\",\"display_name\":\"Benjamin Reed\"},{\"username\":\"ranger\",\"display_name\":\"ranger\"}],\"subject\":\"Hey, you're still an asshole.\",\"messages\":\"1 message\",\"timestamp\":\"2015-01-14T17:10:21.575Z\",\"is_unread\":true,\"original_id\":\"seamail-54b6a2fda7e46af7b900000a\"}"
}
```

When I call add() I get a crash in `initWithOptions:dict`:

```
2015-01-14 14:17:04.666 CruiseMonkey[3695:1432114] Property list invalid for format: 200 (property lists cannot contain objects of type 'CFNull')

2015-01-14 14:17:04.668 CruiseMonkey[3695:1432114] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'unable to serialize userInfo: (null)'

*** First throw call stack:

(0x186a6e59c 0x1971c40e4 0x18b793954 0x100109318 0x100102afc 0x1001cce30 0x1001ccdf0 0x1001d99a0 0x1001daf18 0x1979e52e4 0x1979e4fa8)

libc++abi.dylib: terminating with uncaught exception of type NSException
```

Looks like userInfo isn't getting initialized somewhere?  My ObjC-fu is weak.  :)

**Comment by katzer** [comment]
Have fixed it!

**Comment by RangerRick** [comment]
OK, I no longer get a crash, but I still don't see a notification pop up in iOS.  The badge updates, but I never get a banner at the top of my phone, even though I have configured it like that in the notifications settings.

**Comment by katzer** [comment]
You will only see a pop up if the app is in background

**Comment by RangerRick** [comment]
Ah.  Is there a way to do the same type of pop-up message in the foreground?

**Comment by RangerRick** [comment]
Anyways, you solved the crash, marking as closed.  Thanks!

**Comment by katzer** [comment]
Yes thats on the list

**Comment by lunchbag** [comment]
Hm, I am seeing this issue now. How was it fixed? I'm on 0.8.2-dev

**Comment by Avinash-Raj** [comment]
@katzer I got the same error. Link to the Stackoverflow post  http://stackoverflow.com/questions/38369389/ionic-app-crashes-on-ios-once-i-give-permission-to-local-notification.
